content/md/en/docs/main-docs/build/build-process.md

```diff
@@ -1,118 +1,155 @@
 ---
 title: Build process
-description: This article describes what happens under the hood when building and executing a Substrate node.
+description: Describes how a Substrate node is compiled into Rust and WebAssembly binaries and how the binaries are used to execute calls into the Substrate runtime.
```

Comment:
There is not a "rust" binary. There are native binaries, which are compiled for machine code to a specific native machine.

Reply:
Native binary is just an optimization, and something we were able to do since Rust can compile to both a native binary and a wasm binary at nearly zero cost.

The native binary is something we plan to remove soon, and Substrate will only use a Wasm binary. At that point, unfortunately this page will be a whole lot less useful.